corehq/apps/export/views/download.py

```diff
@@ -420,21 +419,16 @@ def prepare_form_multimedia(request, domain):
             'error': _("Please check that you've submitted all required filters."),
         })
 
-    download = DownloadBase()
-    export_object = view_helper.get_export(export_specs[0]['export_id'])
-
+    export = view_helper.get_export(export_specs[0]['export_id'])
     datespan = filter_form.cleaned_data['date_range']
-    mobile_user_and_group_slugs = filter_form.get_mobile_user_and_group_slugs(filter_form_data)
+    user_types = filter_form.get_es_user_types(filter_form_data)
 
+    download = DownloadBase()
     download.set_task(build_form_multimedia_zip.delay(
         domain=domain,
-        startdate=datespan.startdate.isoformat(),
-        enddate=(datespan.enddate + timedelta(days=1)).isoformat(),
-        app_id=export_object.app_id,
-        xmlns=export_object.xmlns if hasattr(export_object, 'xmlns') else '',
-        export_id=export_object.get_id,
-        zip_name='multimedia-{}'.format(unidecode(export_object.name)),
```

Comment:
These 4 params are all pulled from the export, which the task needs anyways, so there's no need to pass them in separately.  I did keep the export object around for an upcoming change.